x-pack/plugins/ml/public/application/components/chart_tooltip/chart_tooltip.tsx

```diff
@@ -29,46 +29,48 @@ const renderHeader = (headerData?: ChartTooltipValue, formatter?: TooltipValueFo
  */
 export const FormattedTooltip: FC<{ tooltipData: TooltipData }> = ({ tooltipData }) => {
   return (
-    <div className="mlChartTooltip">
-      {tooltipData.length > 0 && tooltipData[0].skipHeader === undefined && (
-        <div className="mlChartTooltip__header">{renderHeader(tooltipData[0])}</div>
-      )}
-      {tooltipData.length > 1 && (
-        <div className="mlChartTooltip__list">
-          {tooltipData
-            .slice(1)
-            .map(({ label, value, color, isHighlighted, seriesIdentifier, valueAccessor }) => {
-              const classes = classNames('mlChartTooltip__item', {
-                // eslint-disable-next-line @typescript-eslint/naming-convention
-                echTooltip__rowHighlighted: isHighlighted,
-              });
-
-              const renderValue = Array.isArray(value)
-                ? value.map((v) => <div key={v}>{v}</div>)
-                : value;
-
-              return (
-                <div
-                  key={`${seriesIdentifier.key}__${valueAccessor}`}
-                  className={classes}
-                  style={{
-                    borderLeftColor: color,
-                  }}
-                >
-                  <EuiFlexGroup>
-                    <EuiFlexItem className="eui-textBreakWord mlChartTooltip__label" grow={false}>
-                      {label}
-                    </EuiFlexItem>
-                    <EuiFlexItem className="eui-textBreakAll mlChartTooltip__value">
-                      {renderValue}
-                    </EuiFlexItem>
-                  </EuiFlexGroup>
-                </div>
-              );
-            })}
-        </div>
-      )}
-    </div>
+    <TooltipContainer>
```

Comment:
I think we can remove this, the `mlChartTooltip` already has its own tooltip style applied

Reply:
Done in 8e9dce4